### ISSUE TITLE
Approve Login / Logout Flow

### ISSUE THREAD

**Issue by jclangst** [silentshop/react#13]

**Comment by tylerthedeveloper** [comment]
Login in fine, we still need to add back create account

**Comment by jclangst** [comment]
Login is the same as sign up. There is no need for a separate sign up button. We can send users through a different interface if they are logging in for the first time if you send me the wireframes of what you are envisioning.

**Comment by tylerthedeveloper** [comment]
how are they the same?

If we are coercing a new stellar account to be made then they will enter in their personal user information to fill their profile, not just the key. I will get you a wireframe.

**Comment by jclangst** [comment]
I think we might still not be on the same page about this. I thought we had reached the understanding that we are never creating or managing stellar accounts on the server side. Managing and storing secret key information is a cardinal sin in the crypto world hence why I spent significant time developing a system that allows all of the crypto interface to stay strictly on the client side.

If you still want to manage people's wallets on the server side (like how crypto exchanges work), you are taking on significant liability and will have to follow all of the applicable data security, disclosure, and KYC regulations. Moreover, unless you have this all scoped under a limited liability entity and I have a formal contract with such an entity, I cannot work on developing that part of the system because I would be exposing myself to personal liability should you mishandle sensitive information in the future (like putting it in a database that isn't protected by the latest ISO standards). Finally, I really think this will be a black mark on your platform when it comes to marketing it -- as a crypto user, I would never use your service if you force me to turn over control of my funds to you; that is the antithesis to the entire mantra of decentralization and personalized security.

If you still want to go ahead with that, I can hook up whatever serverside stellar interface you develop into the react application, but I cannot be party to developing and maintaining such an interface.
